ads/nativery.js

```diff
@@ -40,6 +41,19 @@ export function nativery(global, data) {
     }
   });
 
+  // install observation to check if is in viewport
+  var unlisten = global.context.observeIntersection(function(changes) {
+    changes.forEach(function(c) {
+      global._nativery.visible = parseInt(
```

Comment:
Why using `parseInt` when the argument is not a string? Perhaps you want to do rounding down?

Reply:
Argument is float. Keep parseInt is a problem?

Reply:
`Math.floor` has been widely supported (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor), and seems to fit better to the scene.

Reply:
Resolved